src/common/user-store.ts

```diff
@@ -89,6 +92,23 @@ export class UserStore extends BaseStore<UserStoreModel> {
     return semver.gt(getAppVersion(), this.lastSeenAppVersion);
   }
 
+  @action
+  setTableSizing(clusterId: string, tableId: string, sizes: number[]) {
```

Comment:
Actually, this really isn't a setting because you don't change it in the user settings. This should utilize #2279

Reply:
Noticed this too, working on it